### DISCUSSION TITLE
Minimum Build Requirement Not Met

### DISCUSSION THREAD

**Discussion by 4rc0s** [zigtools/zls#1501] [Q&A] [answered]
I'm getting started with Zig and trying to install zls to use with vscode. I installed zig itself using homebrew, which currently provides v0.11.0. When I try and build zls, I get this error:

```
 error: Your Zig version v0.11.0 does not meet the minimum build requirement of v0.12.0-dev.722+412d863ba
            @compileError(std.fmt.comptimePrint("Your Zig version v{} does not meet the minimum build requirement of v{}", .{ current_zig, min_zig }));
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
referenced by:
    runBuild__anon_6791: /usr/local/Cellar/zig/0.11.0/lib/zig/std/Build.zig:1639:37
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
```
 Any thoughts on how to fix, or do I need to wait until 0.12.0 is available via homebrew?

**Comment by llogick** [comment]
You could [grab a "nightly"](https://ziglang.org/download/) .

**Comment by techatrix** [comment, marked as answer]
Here are your options:
- install ZLS 0.11.0 with [homebrew](https://formulae.brew.sh/formula/zls)
- build ZLS 0.11.0 instead of master
- instead of using homebrew to get zig 0.11.0, you can instead install the [latest build](https://ziglang.org/download/) of zig and then build ZLS master from source.
